sql/updates/world/4.3.4/2021_07_23_00_world.sql

```diff
@@ -0,0 +1,360 @@
+REPLACE INTO `creature_template` VALUES (44604, 0, 0, 0, 0, 0, 33992, 0, 0, 0, 'Forsaken Trooper', '', '', '', 0, 13, 13, 0, 0, 68, 0, 1, 1.14286, 1, 0, 0, 2000, 2000, 1, 1, 1, 0, 2048, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, '', 0, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 1, 0, 0, 0, '', 15595);
+REPLACE INTO `creature_template` VALUES (44605, 0, 0, 0, 0, 0, 33993, 0, 0, 0, 'Forsaken Trooper', '', '', '', 0, 13, 13, 0, 0, 68, 0, 1, 1.14286, 1, 0, 0, 2000, 2000, 1, 1, 1, 0, 2048, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, '', 0, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 1, 0, 0, 0, '', 15595);
+REPLACE INTO `creature_template` VALUES (44606, 0, 0, 0, 0, 0, 33994, 0, 0, 0, 'Forsaken Trooper', '', '', '', 0, 13, 13, 0, 0, 68, 0, 1, 1.14286, 1, 0, 0, 2000, 2000, 1, 1, 1, 0, 2048, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, '', 0, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 1, 0, 0, 0, '', 15595);
+
+REPLACE INTO `creature_template_addon` VALUES (44608, 0, 0, 0, 50331648, 1, 3, '');
+REPLACE INTO `creature_template_addon` VALUES (44609, 0, 0, 0, 50331648, 1, 3, '');
+REPLACE INTO `creature_template_addon` VALUES (44610, 0, 0, 0, 50331648, 1, 3, '');
+REPLACE INTO `creature_template_addon` VALUES (44615, 0, 0, 0, 0, 257, 3, '83231');
+REPLACE INTO `creature_template_addon` VALUES (44592, 0, 0, 0, 0, 1, 0, '80636');
+REPLACE INTO `creature_template_addon` VALUES (44593, 0, 0, 0, 0, 1, 0, '80636');
+REPLACE INTO `creature_template_addon` VALUES (44825, 0, 0, 0, 0, 257, 0, '');
+REPLACE INTO `creature_template_addon` VALUES (44821, 0, 0, 0, 0, 1, 0, '');
+REPLACE INTO `creature_template_addon` VALUES (44882, 0, 0, 0, 0, 1, 0, '');
+REPLACE INTO `creature_template_addon` VALUES (44893, 0, 0, 0, 0, 1, 0, '');
+REPLACE INTO `creature_template_addon` VALUES (44894, 0, 0, 0, 0, 1, 0, '');
+REPLACE INTO `creature_template_addon` VALUES (44884, 0, 0, 0, 0, 1, 0, '');
+REPLACE INTO `creature_template_addon` VALUES (44883, 0, 0, 0, 0, 1, 0, '');
+REPLACE INTO `creature_template_addon` VALUES (44632, 0, 0, 0, 0, 2, 3, '');
+REPLACE INTO `creature_template_addon` VALUES (44764, 0, 0, 0, 0, 1, 4, '');
+REPLACE INTO `creature_template_addon` VALUES (44731, 0, 0, 0, 0, 1, 4, '');
+
+REPLACE INTO `creature_template_movement` VALUES (44610, 1, 0, 1, 0, 0, NULL);
+REPLACE INTO `creature_template_movement` VALUES (44608, 1, 0, 1, 0, 0, NULL);
+REPLACE INTO `creature_template_movement` VALUES (44609, 1, 0, 1, 0, 0, NULL);
+
+INSERT INTO `creature_text` VALUES (44365, 0, 0, 'Where is that ogre-headed buffoon?', 12, 0, 100, 6, 0, 20459, 0, 0, 0, 'VO_QE_SP_Sylvanas_SPEvent01');
```

Comment:
delete from first, and no bazillion single inserts. do one insert query for all texts at once.

Reply:
Fixed.